src/librustdoc/html/render/print_item.rs

```diff
@@ -1415,3 +1423,42 @@ fn document_non_exhaustive(w: &mut Buffer, item: &clean::Item) {
         w.write_str("</div>");
     }
 }
+
+fn document_ty_layout(w: &mut Buffer, cx: &Context<'_>, ty_def_id: DefId) {
```

Comment:
Should probably rename this to `document_type_layout` so that it's consistent with the name of the CLI flag.